wasp/draw565.py

```diff
@@ -355,3 +355,52 @@ def line(n):
             chunks.append(end)
 
         return chunks
+
+    def line(self, x0=0, y0=0, x1=0, y1=0, color=0xffff):
+        """Draw a line between points (x0;y0) and (x1;y1) with color.
+
+        Example:
+
+        .. code-block:: python
+
+            draw = wasp.watch.drawable
+            draw.line(0, 120, 240, 240, 0xf800)
+
+        :param x0: X coordinate of the start of the line, defaults to 0
+        :param x1: Y coordinate of the end of the line, defaults to 0
+        :param y0: Y coordinate of the start of the line, defaults to 0
+        :param y1: Y coordinate of the end of the line, defaults to 0
+        :param color: Color to draw line in, in RGB565 format, defaults to 0xffff
+        """
+
+        px = ((color >> 8) & 0xFF, color & 0xFF)
```

Comment:
Why is the bytes missing? This should be: px = bytes(((color >> 8) & 0xFF, color & 0xFF)) 

Has this code been tested on a real device?

Reply:
well this is embarrassing O.O

I'll fix it and test the fix again later today

Reply:
No worries.

The simulator will never cover everything a real device does but, as it happens, in this case I have been able to modify it to allow it to pick up these kind of problems: https://github.com/daniel-thompson/wasp-os/commit/e3b2c7bf7d7a4cc7dc2cafc28f35e30f44260a21

Reply:
resolved, thanks

Reply:
waiting for CI for a build I can test on my pinetime, will update with results.